### ISSUE TITLE
FPFH descriptor and Deep Learning Using Tensorflow

### ISSUE THREAD

**Issue by gledsonmelotti** [intel-isl/Open3D#2659]
Hi guys, I'm sorry for the inconvenience. I would like to understand how to use open3D codes with deep learning (tensorflow). Can anybody help me? An example below:
```
def fpfh(vector):
    lf = []
    x,bat=vector
    for LF in range(bat):
        Cloud=x[LF]
        pcd = o3d.geometry.PointCloud()
        pcd.points = o3d.utility.Vector3dVector(Cloud)
        #np.asarray(pcd.points)
        voxel_size = 0.05 # means 5cm for the dataset
        radius_normal = voxel_size * 2
        pcd.estimate_normals(o3d.geometry.KDTreeSearchParamHybrid(radius=radius_normal, max_nn=30))
        radius_feature = voxel_size * 5
        pcd_fpfh = o3d.pipelines.registration.compute_fpfh_feature(pcd,o3d.geometry.KDTreeSearchParamHybrid(radius=radius_feature, max_nn=100))
        localfeature = pcd_fpfh.data
        localfeature = StandardScaler().fit_transform(localfeature)
        localfeature = localfeature.T
        lf.append(localfeature)
    xlf=np.array(lf)
    return tf.convert_to_tensor(xlf)

def conv_bn(x, filters):
    x = Conv1D(filters, kernel_size=1, kernel_regularizer=l2(1e-4), use_bias = True, bias_regularizer=l2(1e-4), padding="valid")(x)
    x = BatchNormalization(momentum=0.0)(x)
    return Activation("relu")(x)

def dense_bn(x, filters):
    x = Dense(filters, kernel_regularizer=l2(1e-4), use_bias = True, bias_regularizer=l2(1e-4))(x)
    x = BatchNormalization(momentum=0.0)(x)
    return Activation("relu")(x)

def tnet(inputs, num_features):
    bias = keras.initializers.Constant(np.eye(num_features).flatten())
    x = conv_bn(inputs, 32)
    x = conv_bn(x, 64)
    x = conv_bn(x, 512)
    x = GlobalMaxPooling1D()(x)
    x = dense_bn(x, 256)
    x = dense_bn(x, 128)
    x = Dense(
        num_features * num_features,
        kernel_regularizer=l2(1e-4),
        use_bias = True,
        kernel_initializer="GlorotNormal",
        bias_initializer=bias,
        activity_regularizer=l2(1e-4)
        )(x)
    feat_T = Reshape((num_features, num_features))(x)
    # Apply affine transformation to input features
    return Dot(axes=(2, 1))([inputs, feat_T])

input_pc = Input(shape=(num_points, 3),name='entrada')
x = tnet(input_pc, 3) # T-net layers.
bat = x.shape[0]
xlf = tf.keras.layers.Lambda(function=fpfh)([x,bat])

**Comment by 1939938853** [comment]
Good to see some open3d deep learning application examples.

Thanks
